Configuration/StandardSequences/python/L1TrackTrigger_cff.py

```diff
@@ -5,8 +5,13 @@
 from L1Trigger.TrackerDTC.ProducerED_cff import *
 from L1Trigger.TrackFindingTracklet.L1HybridEmulationTracks_cff import *
 
-L1TrackTrigger=cms.Sequence(TrackTriggerClustersStubs*TrackTriggerAssociatorClustersStubs*TrackerDTCProducer*L1PromptExtendedHybridTracksWithAssociators)
-#L1TrackTrigger=cms.Sequence(TrackTriggerClustersStubs*TrackTriggerAssociatorClustersStubs*TrackerDTCProducer*L1PromptExtendedHybridTracks)
+L1TrackTrigger=cms.Sequence(TrackTriggerClustersStubs*TrackTriggerAssociatorClustersStubs*TrackerDTCProducer)
 
+# Customisation to enable TTTracks in geometry D41 and later (corresponding to phase2_hgcalV10 or later). Includes the HGCAL L1 trigger
+_tttracks_l1tracktrigger = L1TrackTrigger.copy()
+_tttracks_l1tracktrigger.add(L1PromptExtendedHybridTracksWithAssociators)
+
+from Configuration.Eras.Modifier_phase2_hgcalV10_cff import phase2_hgcalV10
```

Comment:
@silviodonato 
You are using `phase2_hgcalV10` to identify D41 or later geometries, to switch on the TrackFinder, i.e. include it in the `L1TrackTrigger` sequence.

Perhaps it would would be more pedestrian to introduce a `phase2_trackerV14` modifier, 
 add it to `Phase2C8` and then use it here to make a switch.

Reply:
Yes, indeed, adding `phase2_trackerV14` is the proper way to do it. @kpedro88  do you agree to add the modifier `phase2_trackerV14`?

Reply:
Then I will use it to hack my 'L1' sequence to include track-based producers.  
Note, I am not happy with this solution for a long run, and I consider it only as a temporary work-around solution.  There is no reason Hybrid TrackTrigger could not be made to

Reply:
Yes, that seems fine to me

Reply:
ok.  will proceed that way in the PR.